keras_cv/conftest.py

```diff
@@ -58,6 +64,10 @@ def pytest_collection_modifyitems(config, items):
     skip_extra_large = pytest.mark.skipif(
         not run_extra_large_tests, reason="need --run_extra_large option to run"
     )
+    skip_tf_only = pytest.mark.skipif(
```

Comment:
Ok that answers my question. I think `tf_keras_only` might be more explicit, TF could be MBK + TF

Reply:
Renamed to `tf_keras_only`

Reply:
I think that fits. I make heavy use of a `tf_only` marker in KerasNLP that means run on MBK + TF or tf.keras.